### PR TITLE
Update for 1.13.1

### PR DESCRIPTION
**Pull request by Brokkonaut**
I updated the plugin for Bukkit/Spigot 1.13.

- Materials instead of material ids
- Use the new passenger API
- Build against java 1.8 instead of 1.6
- Some minor cleanups
- Update version to 1.9.0 because this is a breaking change (will only work on Spigot 1.13+ because of the new material names)

### CODE REVIEW

**Comment by Brokkonaut** [comment]
Thanks, I changed that

File: src/main/java/com/wimbli/WorldBorder/BorderData.java [hunk 1]
```diff
@@ -291,26 +291,102 @@ public Location correctedPosition(Location loc)
...
+		safeOpenBlocks.add(Material.STONE_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.OAK_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.SPRUCE_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.BIRCH_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.JUNGLE_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.ACACIA_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.DARK_OAK_PRESSURE_PLATE);
+		safeOpenBlocks.add(Material.REDSTONE_TORCH);
+		safeOpenBlocks.add(Material.REDSTONE_WALL_TORCH);
+		safeOpenBlocks.add(Material.STONE_BUTTON);
```
Comment (Black-Hole) [inline src/main/java/com/wimbli/WorldBorder/BorderData.java]: Maybe add the wood buttons, too?

Comment (Brokkonaut) [reply src/main/java/com/wimbli/WorldBorder/BorderData.java]: Good idea, I had just transformed the old list, but now I added some new blocks that were not in that list.

**Comment by Brokkonaut** [comment]
Updated for Bukkit/Spigot 1.13.1

**Pull request merged**